src/remote_gym/remote_environment.py

```diff
@@ -11,6 +12,34 @@
 from gymnasium.spaces import Box, Discrete, MultiDiscrete, Space
 
 
+class RemoteArgs(TypedDict):
+    """
+    Configuration on how to start the remote environment.
+
+    Attributes:
+        repo (Optional[str]): The repository to clone. If set to None, cloning is not permitted.
+        reference (Optional[str]): The tag, branch, or commit ID to check out.
+        entrypoint (str): The filename of the entrypoint, containing a create_environment function.
+        kwargs (Dict[str, any]): Additional parameters passed to the create_environment function.
+
+
+    Example:
+        ```
+        args = RemoteArgs(
+            repo='https://github.com/example/repo.git',
+            reference='v1.0',
+            entrypoint='main.py',
+            kwargs={'param1': 'value1', 'param2': 2}
```

Comment:
Maybe we can call it something more understandable like `env_init_kwargs` or `entrypoint_kwargs`.